AlfrescoApp/Model/Managers/RealmSyncManager.m

```diff
@@ -1282,7 +1286,10 @@ - (void)reachabilityChanged:(NSNotification *)notification
         self.lastConnectivityFlag = hasInternetConnection;
         if(hasInternetConnection)
         {
-            [self refreshWithCompletionBlock:nil];
+            if(([AccountManager sharedManager].allAccounts.count > 0) && ([AccountManager sharedManager].selectedAccount))
```

Comment:
The first conditions in the if statement seems to be unnecessary. If there are no accounts, you can't have a selected account.